docs/spec/staking/state.md

```diff
@@ -29,10 +29,10 @@ overall functioning of the stake module.
 
 ```golang
 type Params struct {
-    InflationRateChange sdk.Rat // maximum annual change in inflation rate
-	InflationMax        sdk.Rat // maximum inflation rate
-	InflationMin        sdk.Rat // minimum inflation rate
-	GoalBonded          sdk.Rat // Goal of percent bonded atoms
+    InflationDeceChange sdk.Dec // maximum annual change in inflation rate
```

Comment:
`Dece` strikes again!

Reply:
Looked through everything `Dece` and made fixes